auth/src/main/java/org/aerogear/mobile/auth/AuthHeaderProvider.java

```diff
@@ -0,0 +1,37 @@
+package org.aerogear.mobile.auth;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.aerogear.mobile.auth.user.UserPrincipal;
+import org.aerogear.mobile.core.http.interceptors.HeaderProvider;
+
+/**
+ * Default provider class that will build headers based on currently logged in user access token
+ */
+public class AuthHeaderProvider implements HeaderProvider {
+
+    public static final String TYPE = "Bearer ";
+    public static final String HEADER_KEY = "Authorization";
```

Comment:
HEADER should be removed or add also in TYPE

Reply:
You mean `HEADER_KEY` => `_KEY` => `TYPE_KEY`?